.github/workflows/cpu-tests.yml

```diff
@@ -25,7 +25,10 @@ jobs:
       matrix:
         include:
           - {os: "macOS-11", python-version: "3.10"}
+          - {os: "ubuntu-20.04", python-version: "3.11"}
```

Comment:
Out of curiosity: why 20.04 and not the latest LTS — 22.04?

Reply:
Good point

```suggestion
          - {os: "ubuntu-22.04", python-version: "3.11"}
```

Reply:
What about others? 
Or you want the latest LTS use only for the Python 3.11?